ElectronClient/theme.js

```diff
@@ -1,6 +1,12 @@
 const Setting = require('lib/models/Setting.js');
+const lightStyle = require('./gui/style/theme/light'); // Default theme
+const darkStyle = require('./gui/style/theme/dark');
+const draculaStyle = require('./gui/style/theme/dracula');
+const solarizedLightStyle = require('./gui/style/theme/solarizedLight');
+const solarizedDarkStyle = require('./gui/style/theme/solarizedDark');
 const nordStyle = require('./gui/style/theme/nord');
```

Comment:
How about moving all these themes under a `themes` object? Like `const theme = {[Setting.THEME_LIGHT]: require('./gui/style/theme/light'), ...`
That will make it easier to add new theme and we'll automatically have the mapping theme ID to module.

Reply:
Sure, I can do that if you prefer. I didn't do it since this was how the nord theme was added so I figured I'd just stay on the same style. Should be able to do this soon almost directly.

Reply:
This was the first time I tried doing something like this, so this might not be how you meant it? Feel free to let me know and I'll change it again if needed! (I've only changed CSS stuff before).